benchmarks/src/test/java/org/apache/druid/benchmark/query/SqlBenchmark.java

```diff
@@ -557,7 +557,15 @@ public void setup()
           schemaInfo,
           DimensionsSpec.builder().setDimensions(columnSchemas).build(),
           TransformSpec.NONE,
-          IndexSpec.DEFAULT,
+          new IndexSpec(
+              null,
+              null,
+              getStringEncodingStrategy(),
+              null,
+              null,
+              null,
+              null
+          ),
```

Comment:
For brevity:
```suggestion
          IndexSpec.builder().withStringDictionaryEncoding(getStringEncodingStrategy()).build(),
```

Reply:
Cool idea, I just committed this change.